frontend/comfywatchman-dashboard/src/index.css

```
@@ -0,0 +1,2975 @@
+    -webkit-text-decoration: underline dotted;
+    text-decoration: underline dotted;
+  }
+
+  h1, h2, h3, h4, h5, h6 {
+    font-size: inherit;
+    font-weight: inherit;
+  }
+
+  a {
+    color: inherit;
+    -webkit-text-decoration: inherit;
```

Stylelint (17.11.1)

[error] Duplicate property "-webkit-text-decoration" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)